### DISCUSSION TITLE
Does SeisSol support local implicit?

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#897] [Q&A] [answered]
As far as I know, SeisSol is a time domain explicit solver, does SeisSol support local implicit solving?

**Comment by sebwolf-de** [comment, marked as answer]
Yes. For the poroelastic material model, we have implemented a locally implicit space time predictor: https://www.sciencedirect.com/science/article/abs/pii/S0021999121007816

**Reply by smallriver666** [reply]
Thank you !
